solr/solrj/src/java/org/apache/solr/common/params/CollectionAdminParams.java

```diff
@@ -49,6 +49,8 @@ public interface CollectionAdminParams {
 
   String REPLICATION_FACTOR = "replicationFactor";
 
+  String REPLICA_TYPE = "type";
```

Comment:
There is already a `REPLICA_TYPE` in `ZkStateReader`

Reply:
Yes, but it is not accessible from solrj.

Reply:
Done.